docs(dedibox): update faq (#1184)
scaleway/docs-content: faq/dedibox-in-scaleway-console.mdx

---
meta:
  title: Dedibox in Scaleway console
  description: Discover how to link your Dedibox servers to your Scaleway account.
content:

>   h1: Dedibox in Scaleway console

Reviewer: ```suggestion
  h1: Dedibox in the Scaleway console
```

## What impact will linking accounts have on my billing?

You will continue to receive an invoice for each type of product (one for Scaleway Dedibox and one for Scaleway Cloud products).

> However, the default payment method defined in the Scaleway console will allow you to pay all your Scaleway invoices (Scaleway Dedibox and Scaleway Cloud). It is important that you verify the default payment method defined in the Scaleway console after the linking process, to ensure the proper payment method will be used for your future invoices.<br />

Reviewer 2: However, it is the payment method defined in your Scaleway console that will allow you to pay all your Scaleway invoices (Scaleway Dedibox and Scaleway Cloud). It is thus important that following the linking process, you verify whether the default payment method defined in the Scaleway console is your preferred payment method for future invoices.

Author: ```suggestion
However, it is the payment method defined in your Scaleway console that will allow you to pay all your Scaleway invoices (Scaleway Dedibox and Scaleway Cloud). It is thus important that following the linking process, you verify whether the default payment method defined in the Scaleway console is your preferred payment method for future invoices.<br />
```

## What are the terms of the voucher offered for the migration?

> For the first customers following the linking process, a discount of 100€ is applied directly to their Scaleway account. This discount is valid for six months after the link of your accounts. It is available on our Scaleway cloud services ordered through the Scaleway console, excluding Dedibox and Elastic Metal products.

Reviewer 2: The first customers to follow the linking process will be granted a discount of 100€. This discount is directly applied to the Scaleway account and is valid for six months following the linking process. It is available on Scaleway cloud services ordered through the Scaleway console and does not include Dedibox and Elastic Metal products.

Author: ```suggestion
The first customers to follow the linking process will be granted a discount of 100€. This discount is directly applied to the Scaleway account and is valid for six months following the linking process. It is available on Scaleway cloud services ordered through the Scaleway console and does not include Dedibox and Elastic Metal products.
```

> ## How can I ask my remaining questions?

Reviewer 2: Who can I ask if I have further questions?

Author: ```suggestion
## Who can I ask if I have further questions?
```
